packages/flutter_test/lib/src/controller.dart

```diff
@@ -1096,36 +1096,42 @@ abstract class WidgetController {
   /// Given a widget `W` specified by [finder] and a [Scrollable] widget `S` in
   /// its ancestry tree, this scrolls `S` so as to make `W` visible.
   ///
-  /// Usually the `finder` for this method should be labeled
-  /// `skipOffstage: false`, so that [Finder] deals with widgets that's out of
-  /// the screen correctly.
+  /// Usually the `finder` for this method should be labeled `skipOffstage:
+  /// false`, so that the [Finder] deals with widgets that are off the screen
+  /// correctly.
   ///
-  /// This does not work when the `S` is long and `W` far away from the
-  /// displayed part does not have a cached element yet. See
-  /// https://github.com/flutter/flutter/issues/61458
+  /// This does not work when `S` is long enough, and `W` far away enough from
+  /// the displayed part does of `S`, that `S` has not yet cached `W`'s element
+  /// yet. Consider using [scrollUntilVisible] in such a situation.
   ///
-  /// Shorthand for `Scrollable.ensureVisible(element(finder))`
+  /// See also:
+  ///
+  ///  * [Scrollable.ensureVisible], which is the production API used to
+  ///    implement this method.
   Future<void> ensureVisible(Finder finder) => Scrollable.ensureVisible(element(finder));
 
   /// Repeatedly scrolls a [Scrollable] by `delta` in the
-  /// [Scrollable.axisDirection] until `finder` is visible.
+  /// [Scrollable.axisDirection] direction until a widget matching `finder` is
+  /// visible.
   ///
-  /// Between each scroll, wait for `duration` time for settling.
+  /// Between each scroll, advances the clock by `duration` time.
   ///
-  /// If `scrollable` is `null`, this will find a [Scrollable].
+  /// Scrolling is performed until the start of the `finder` is visible. This is
+  /// due to the default parameter values of [Scrollable.ensureVisible] method.
```

Comment:
nit: ... of _the_ [Scrollable.ensureVisible] method.

Reply:
fixed